.eslintrc

```diff
@@ -2,5 +2,14 @@
   "extends": "standard",
   "rules": {
     "comma-dangle": [0]
+  },
+  "globals": {
+    "describe": true,
+    "expect": true,
+    "it": true,
+    "before": true,
+    "after": true,
+    "StemSkeleton": true,
```

Comment:
We have StemSkeleton global during tests?

Reply:
Not actually. Tests in `organic-stem-devtools` seem to have it, so I guess it's a leftover from it.
Removed it.